draft-ietf-httpbis-rfc6265bis.md

```diff
@@ -1642,8 +1650,8 @@ user agent MUST process the cookie as follows:
 
     1.  Let the domain-attribute be the empty string.
 
-8.  If the domain-attribute contains a character that is not in the range of {{USASCII}}
-    characters, abort this algorithm and ignore the cookie entirely.
+8.  If the domain-attribute contains a character that is not in CHAR, abort
+    this algorithm and ignore the cookie entirely.
```

Comment:
Addresses

> "[USASCII]" doesn't seem to be strictly defined.  E.g. the term is
used descriptively in sec 2.2 but a definition is not explicitly
imported from RFC 5234, and 5234 doesn't use the term.  Probably what
is meant is "is not in CHAR", which is explicitly defined.